08 - FractorCalculator.java

```diff
@@ -0,0 +1,103 @@
+import java.io.*;
+   import java.util.Scanner;
+  public class FractionCalculator{
+     public  class Fraction{
+                  private int nomo;
+                  private int deno;
+                  Fraction(){nomo=0;deno=1;}
+                  Fraction(int n,int d){nomo=n;deno=d;}
+  
+     }
+
+          private static Fraction f1;
+          private static Fraction f2;
+ 
+
+     public static void main(String[] args){
+         int choi;
+       Scanner sc=new Scanner(System.in);
+       FractionCalculator FC=new FractionCalculator();
+       Fraction f;
+       printMenu();
+           System.out.print("Choose one operation:");
+       choi=sc.nextInt();
+       switch(choi){
+             case 1: FC.getInput();
+                           f=FC.addFraction(f1,f2);
+                           System.out.format("\nResult:%d/%d\n",f.nomo,f.deno);
+                           break;
+               case 2: FC.getInput();
+                            f=FC.subtractFraction(f1,f2);
+                            System.out.format("\nResult:%d/%d\n",f.nomo,f.deno);
+                            break;
+               case 3: FC.getInput();
+                             f=FC.multiplyFraction(f1,f2);
+                             System.out.format("\nResult:%d/%d\n",f.nomo,f.deno);
+                             break;
+               case 4: FC.getInput();
+                             f=FC.divideFraction(f1,f2);
+                             System.out.format("\nResult:%d/%d\n",f.nomo,f.deno);
+                             break;
+                   default: System.out.println("Invalid choice!");
+         }
+    }
+
+    public Fraction addFraction(Fraction f1,Fraction f2){
+                Fraction resul=new Fraction();
```

Comment:
I like how you have a different method for each operation - each with a clear name as well.